examples/notebooks/tsne_mnist.ipynb

```diff
@@ -0,0 +1,328 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f5c4abc0",
+   "metadata": {},
+   "source": [
+    "# Intel® Extension for Scikit-learn TSNE for MNIST dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "23512089",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from timeit import default_timer as timer\n",
+    "from IPython.display import HTML\n",
+    "from sklearn import metrics\n",
+    "from sklearn.datasets import fetch_openml\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "%matplotlib inline\n",
+    "\n",
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b6e359f6",
+   "metadata": {},
+   "source": [
+    "### Download the data "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "27b99b44",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x, y = fetch_openml(name='mnist_784', return_X_y=True)"
```

Comment:
Data source is not referenced (site/authors).

Reply:
Done.